docs/src/main/asciidoc/_templates/template-tutorial.adoc

```diff
@@ -12,6 +12,7 @@ TODO:
 [id="..."]
 = Title using sentence capitalization
 include::_attributes.adoc[]
+:numbered:
```

Comment:
is this issue of :numbered: not general? 

should it be set in pom.xml attributes section instead?

Reply:
Ah, that's a valid concern, any thoughts @michelle-purcell ?

Reply:
@michelle-purcell could you please respond to the above question?

Reply:
@maxandersen - Sorry, I missed this Q... 

Yes, I think you are right @maxandersen - Adding the header attribute `numbered` fixed the .1. numbering part of the issue but wasn't enough to fix the indentation level 3-4 part of the numbering issue. The latest edits also resolve the numbering format issue too. I just tested without that attribute and the numbering works fine locally so I removed it from the 2 modified topics and the tutorial template. Thanks :star: